app/code/core/Mage/Core/Model/Date.php

```diff
@@ -160,7 +160,7 @@ public function gmtTimestamp($input = null)
      * Input date must be in GMT timezone
      *
      * @param  int|string $input date in GMT timezone
-     * @return int
+     * @return string
```

Comment:
This one actually returns an `int` due to the addition operator at the end. Same thing for some other changes in this class.

Reply:
Interessting. It returns `int`, but even phpstorm says it should be `string`.

![string-int](https://user-images.githubusercontent.com/5022236/193429293-263d121c-febb-45f3-9875-75a65c0ee246.PNG)

I'll revert.

Reply:
Seems to be a missing feature/function ... phpStorm does not recognize the operator ...

`$test = 1 + ''` should return `int|string` ... never noticed this. Thanks!